ci-operator/step-registry/baremetal/lab/upi/conf/network/baremetal-lab-upi-conf-network-ref.yaml

```
@@ -14,6 +14,11 @@ ref:
       default: "192.168.80.0/22"
     - name: INTERNAL_NET_V6_CIDR
       default: "fd99:2222:3456::/64"
+    - name: ipv6_enabled
+      default: "false"
+    - name: ipv4_enabled
+      default: "true"
   documentation: |-
     This step will use the QE baremetal host service to setup the install-config networking section 
-    for Dual Stack BM UPI installations on infrastructure leveraging the self managed network topology.
+    for BM UPI installations on infrastructure leveraging the self managed network topology.
```

is this only BM UPI?
```suggestion
    for BM installations on infrastructure leveraging the self managed network topology.
```

---

Done